docs/user-guide/introduction.rst

```diff
@@ -57,6 +57,10 @@ Platform Support Policy
 CentOS/Red Hat Enterprise Linux (RHEL)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+We currently support Red Hat Enterprise Linux 7 as well as CentOS 7.
+
+Starting with Pulp 2.12, we *no longer* support Red Hat Enterprise Linux 6 or CentOS 6.
```

Comment:
Pulp doesn't support itself, though, right? This is the same wording that was previously here...